policies/policies.go

```diff
@@ -197,6 +196,34 @@ func RunMap(baseDir string, metric string, m map[string]interface{}, holder Metr
 	}
 }
 
+//camelCaseToSnakeCase convert a camelCase string to snake_case
+func camelCaseToSnakeCase(input string) string {
+	if input == "" {
+		return ""
+	}
+
+	snakeCase := make([]rune, 0, len(input))
+	runeArray := []rune(input)
+
+	for i, _ := range runeArray {
+		// Check if two consecutive runes are uppercase, if one uppercase rune followed by a lowercase rune separate by an underscore
+		if unicode.IsUpper(runeArray[i]) && unicode.IsUpper(runeArray[i+1]) {
```

Comment:
I guess this will panic, because you are running out of bounds in i+1 if `I` is the last index